src/renderer/ui/components/settings/ToggleableOption.js

```diff
@@ -17,7 +18,8 @@ class SettingsCheckbox extends Component {
   render() {
     return (
       <Checkbox
-        label={this.props.label}
+        label={[this.props.label, this.props.hintLabel
```

Comment:
This will throw warnings in the console (if it doesn't it should) due to not setting a key on each item in an inline array.

I'd also rather that this ternary expression was pulled out of the return call to a variable assignment